pypeit/specobjs.py

```diff
@@ -409,14 +409,23 @@ def name_indices(self, name):
             msgs.error("The '{0:s}' PYPELINE is not defined".format(self[0].PYPELINE))
         return indx
 
-    def slitorder_objid_indices(self, slitorder, objid):
+    def slitorder_objid_indices(self, slitorder, objid, toler=5):
         """
         Return the set of indices matching the input slit/order and the input objid
-        """
+        
+        Args:
+            slitorder: 
+            objid: 
+            toler: 
+
+        Returns: 
+
+        """ # SHANE Fill this in.
+
         if self[0].PYPELINE == 'Echelle':
             indx = (self.ECH_ORDER == slitorder) & (self.ECH_OBJID == objid)
         elif self[0].PYPELINE == 'MultiSlit':
-            indx = (self.SLITID == slitorder) & (self.OBJID == objid)
+            indx = (np.abs(self.SLITID - slitorder) <= toler) & (self.OBJID == objid)
```

Comment:
I guess you also want to decide if this `toler` is 5, 3, or a parameter. Maybe add a TODO?